src/routes/auth.js

```diff
@@ -91,12 +91,12 @@ router.get(
           return next(err);
         }
         const token = jwt.sign(
-          { id: user.email, roleId: user.roleId },
+          { email: user.email, roleId: user.roleId, userName: user.name,userId:user.id },
           process.env.USER_SECRET,
           { expiresIn: '1h' }
         );
 
-        const redirectUrl = `${FRONTENDURL}?token=${token}`;
+        const redirectUrl = `${FRONTENDURL}/login?token=${token}`;
```

Comment:
Does this mean a user will be redirected to their login after a successful Google authentication?

Reply:
yes , i tried to follow what has get implemented on the fronted where there are many items set on local storage from the login container, but i can change that . I was trying to avoid to much repetition on frontend